modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with the condition type, informing you of the current state of your operator's health. The following definitions cover a list of some common ClusterOperator condition types. Some operators may have additional condition types with operator-specific language, which are not included here. 
+
+Cluster version operator is responsible for collecting these status conditions from cluster operators and represents these is an way which will be easier for OCP administrators to understand the state of the OCP cluster.
+
+//Condition types, as well as additional information about your operator, can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+//[source,terminal]
+//----
+//$ oc get clusterversion -o yaml
+//----
```

Comment:
Did you mean to leave these comments in?

Reply:
Yes - there is debate over this command and we decided to hold off on including it for now. We may revisit it in the future. :)